slapp/explorer/views.py

```diff
@@ -523,11 +528,35 @@ def get_context_data(self, **kwargs) -> dict:
         """Manage context data."""
         context = super().get_context_data(**kwargs)
 
-        region_bbox = {
-            entry["name"]: entry["bounding_box"]
-            for entry in models.Region.objects.annotate(bounding_box=Envelope("geom")).values("bounding_box", "name")
-        }
-        regions = [
+        try:
+            region_kiel = models.Region.objects.get(name="Kiel")
+            region_os = models.Region.objects.get(name="Oderland-Spree")
+        except models.Region.DoesNotExist:
+            region_kiel = None
+            region_os = None
+
+        if region_kiel:
+            ids_kiel = region_kiel.municipality_set.values_list("id", flat=True)
+            context["municipalities_region_kiel"] = municipalities_details(ids_kiel)
+        else:
+            context["municipalities_region_kiel"] = None
+
+        if region_os:
+            ids_os = region_os.municipality_set.values_list("id", flat=True)
+            context["municipalities_region_os"] = municipalities_details(ids_os)
+        else:
+            context["municipalities_region_os"] = None
+
+        regions = self.get_regions_data()
+
+        context["regions"] = regions
+        context["next_url"] = reverse("explorer:esys_robust")
+        return context
+
+    @staticmethod
+    def get_regions_data() -> list:
```

Comment:
As this function is used in other view as well, please move this to an extra module `explorer/regions.py`.